### ISSUE TITLE
merge MMM-Wallpaper with WallberryTheme

### ISSUE THREAD

**Issue by editwentyone** [delightedCrow/WallberryTheme#44]
its would be so awesome to merge the inputs/sources here to wallberrytheme

https://github.com/kolbyjack/MMM-Wallpaper

its able to show much more than unsplash, especially iCloud is very powerful :)

**Comment by Andoramb** [comment]
Yeah, I was just looking if WallberryTheme can display wallpapers from reddit :)

**Comment by mrchips83** [comment]
Would love this, or even better, MMM-BackgroundSlideshow to be added: anyone able to help with it?
I would, but I have no clue about CSS etc.

https://github.com/darickc/MMM-BackgroundSlideshow
